x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/import_rules_utils.ts

```diff
@@ -84,6 +101,52 @@ export const importRules = async ({
               return null;
             }
 
+            if (!allowPrebuiltRules && parsedRule.immutable) {
+              resolve(
+                createBulkErrorObject({
+                  statusCode: 400,
+                  message: i18n.translate(
+                    'xpack.securitySolution.detectionEngine.rules.importPrebuiltRulesUnsupported',
+                    {
+                      defaultMessage:
+                        'Importing prebuilt rules is not supported. To import this rule as a custom rule, remove its "immutable" property try again.',
+                    }
+                  ),
+                  ruleId: parsedRule.rule_id,
+                })
+              );
+
+              return null;
+            }
+
+            if (allowPrebuiltRules) {
+              // TODO how do we differentiate this case from a non-prebuilt import?
```

Comment:
@jpdjere I just wanted to verify this behavior: if a rule does not specify a version, we don't reject the entire request but instead reject the individual rule (regardless of whether the rule represents a prebuilt rule or not [which means  I need to change the error message]).